app/components/Project/Project.js

```diff
@@ -709,6 +717,7 @@ class Project extends Component<Props> {
               <Tab label="People" value="people" classes={tabStyle} />
               <Tab label="Notes" value="projectNotes" classes={tabStyle} />
               <Tab label="Project Log" value="projectLog" classes={tabStyle} />
+              <Tab label="Reproducibility Checklists" value="reproChecklists" classes={tabStyle} />
```

Comment:
Maybe we could shorten to "Checklist", and add "Reproducibility Checklist" as a title on the page?